services/todo-legacy/util/setup.ts

```diff
@@ -0,0 +1,27 @@
+import { datasources } from '../datasources';
+import { TodoController } from '../controllers/todo-controller';
+import { TodoRepository } from '../repositories/todo';
+import { Application } from '@loopback/core';
+
+let Todo = require('../models/todo.json');
+
+/**
+ * Setup your application-level injection here, to minimize clutter. 
+ * 
+ * @export
+ * @param {Application} app
+ */
+export function setBindings(app: Application) {
+  let ds = datasources['ds'];
+  // Controller bindings
+  app.controller(TodoController);
+
+  // Repository and model bindings
+  app.bind('datasources.config').to(ds);
+  app.bind('models.todo').to(Todo);
+  app.bind('repositories.todo').toClass(TodoRepository);
+
+  // Server protocol bindings
+  app.bind('servers.http.enabled').to(true);
+  app.bind('servers.https.enabled').to(true);
+}
```

Comment:
Try to follow recommendations in https://github.com/strongloop/loopback-next-sandbox/tree/master/ts/src for organizing code (put all the above in `application.ts` and start the app in `index.ts`).